egs/tedlium/s5_r3/cmd.sh

```diff
@@ -0,0 +1,27 @@
+# "queue.pl" uses qsub.  The options to it are
+# options to qsub.  If you have GridEngine installed,
+# change this to a queue you have access to.
+# Otherwise, use "run.pl", which will run jobs locally
+# (make sure your --num-jobs options are no more than
+# the number of cpus on your machine.
+
+# Run locally:
+#export train_cmd=run.pl
+#export decode_cmd=run.pl
+
+# JHU cluster (or most clusters using GridEngine, with a suitable
+# conf/queue.conf).
+export train_cmd="queue.pl"
+export decode_cmd="queue.pl --mem 4G"
+
+host=$(hostname -f)
```

Comment:
Please remove the stuff from this line from this file.  We really shouldn't be putting these cluster-specific things in these files.  (I know this was copied from elsewhere).

Reply:
Sorry about that, copied it from s5_r2, you might want to remove it from there as well.